queue/constants.go

```diff
@@ -1,5 +1,16 @@
 package queue
 
+type WorkerType string
+
+const (
+	Publisher WorkerType = "publisher" // indicates that the queue is being injected into a worker which publishes to this queue
```

Comment:
```suggestion
WorkerTypePublisher = "publisher" // indicates that the queue is being injected into a worker which publishes to this queue
```
same way for the other constants as well